### PR TITLE
Added docker image and script

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: scripts/run-observer.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+fi
+
+#check if 'jq' is installed
+if ! command -v jq &> /dev/null; then
+    apt install -y jq
+fi
+
+
+check_nonce_grows() {
+  echo "node is running...wait for the tree to be synced"
```
Comment (iulianpascalau) [inline scripts/run-observer.sh:34]: Pleas move this on L37 as to align with the rest of the repos

File: scripts/run-observer.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    if [ "${CURRENT_NONCE}" -gt 0 ]; then
+      # check if is first time when current nonce is greater than zero
+      if [ "${FIRST_TIME}" -eq 0 ]; then
+        FIRST_TIME=1
+        PREVIOUS_NONCE=${CURRENT_NONCE}
+        sleep 4s
+        continue
+      fi
+      if [ "${CURRENT_NONCE}" -gt "${PREVIOUS_NONCE}" ]; then
+        echo "node is syncing... nonce is growing"
```
Comment (iulianpascalau) [inline scripts/run-observer.sh:56]: ...is increasing

File: scripts/run-observer.sh (hunk 1 above)
Comment (miiu96) [reply scripts/run-observer.sh]: done.

File: scripts/run-observer.sh (hunk 2 above)
Comment (miiu96) [reply scripts/run-observer.sh]: done.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**